cilium-dbg/cmd/bpf_ipcache_get_test.go

```diff
@@ -71,12 +66,11 @@ func (s *BPFIPCacheGetSuite) TestGetLPMValue(c *C) {
 
 	for _, tt := range tests {
 		v, exists := getLPMValue(mustParseIP(tt.ip), entries)
-
-		c.Assert(exists, Equals, tt.hasIdentity, Commentf("No identity was found for ip '%s': wanted '%s'", tt.ip, tt.identity))
+		require.Equal(t, exists, tt.hasIdentity, fmt.Sprintf("No identity was found for ip '%s': wanted '%s'", tt.ip, tt.identity))
 
 		if exists {
 			identity := v.([]string)
-			c.Assert(identity, checker.DeepEquals, tt.identity, Commentf("Wrong identity was retrieved for ip %s", tt.ip))
+			require.Equal(t, len(identity), len(tt.identity), fmt.Sprintf("Wrong number of identities was retrieved for ip %s", tt.ip))
```

Comment:
Shouldn't we compare the slices' content here as well?

Reply:
thanks, it's addressed in https://github.com/cilium/cilium/pull/32210